src/extractor/guidance/motorway_handler.cpp

```diff
@@ -254,7 +259,15 @@ Intersection MotorwayHandler::fromMotorway(const EdgeID via_eid, Intersection in
                     if (road.angle == continue_angle)
                     {
                         road.instruction = getInstructionForObvious(
-                            intersection.size(), via_eid, isThroughStreet(1, intersection), road);
+                            intersection.size(),
+                            via_eid,
+                            ::osrm::extractor::guidance::isThroughStreet(1,
+                                                                         intersection,
+                                                                         node_based_graph,
+                                                                         node_data_container,
+                                                                         name_table,
+                                                                         street_name_suffix_table),
+                            road);
```

Comment:
I think we are in the `osrm::extractor::guidance` namespace already and you can just remove the namespace prefix.

https://github.com/Project-OSRM/osrm-backend/blob/c1664a65d326e87797552fec6bf5fdaddc2fdf5e/src/extractor/guidance/motorway_handler.cpp#L17-L21